src/patterns/organisms/navigation/navigation.hbs

```diff
@@ -17,7 +18,7 @@ links:
 <div class="mzp-c-navigation">
   <div class="mzp-c-navigation-l-content">
     <div class="mzp-c-navigation-container">
-      <button class="mzp-c-navigation-menu-button" type="button" aria-controls="{{ id }}">Menu</button>
+      <button class="mzp-c-navigation-menu-button{{#if button-label}} has-label{{/if}}" type="button" aria-controls="map-c-navigation-items">{{#if button-label}}{{ button-label }}{{ else }}Menu{{/if}}</button>
```

Comment:
Is it OK to set `aia-controls` to point to a class? I thought it needed an ID?

Reply:
Not only is it not okay... I miss spelled it. 🤦‍♀️ 

✅